frontend/templates/work.html

```diff
@@ -179,11 +179,20 @@ <h4>Related Works</h4>
             </div>
         </div>
         <div class="bookSidebar">
+            <div class="bookRead">
```

Comment:
We're going to need some logic to suppress the button if no usable ebook. Same with donate and download buttons